src/app/fddev/configure/genesis.c

```diff
@@ -192,7 +192,7 @@ create_genesis( config_t * const config,
 
   fd_features_t features[1];
   fd_features_disable_all( features );
-  fd_features_enable_hardcoded( features );
+  fd_features_enable_hardcoded( features, 1900 );
```

Comment:
Maybe we should make a #define for what the "default" cluster version is...

Reply:
added `FD_FEATURES_CLUSTER_VERSION`